code/go/0chain.net/smartcontract/storagesc/handler.go

```diff
@@ -215,17 +205,160 @@ func (ssc *StorageSmartContract) GetBlobbersHandler(
 	}
 
 	var sns storageNodesResponse
-	for _, blobber := range blobbers {
-		sn, err := blobberTableToStorageNode(blobber)
-		if err != nil {
-			return ssc.GetBlobbersHandlerDeprecated(ctx, params, balances)
-		}
-		sns.Nodes = append(sns.Nodes, sn)
+	sns.Nodes = make([]storageNodeResponse, len(blobbers))
+	for i, blobber := range blobbers {
+		sn := blobberTableToStorageNode(blobber)
+		sns.Nodes[i] = sn
 	}
 	return sns, nil
 }
 
-func (msc *StorageSmartContract) GetTransactionByHashHandler(
+// GetAllocationBlobbersHandler returns list of all blobbers alive that match the allocation request.
+func (ssc *StorageSmartContract) GetAllocationBlobbersHandler(
+	ctx context.Context,
+	params url.Values, balances cstate.StateContextI) (interface{}, error) {
+	if balances.GetEventDB() == nil {
```

Comment:
Fixing the [blobber table](https://github.com/0chain/0chain/blob/alloc_blob_stable/code/go/0chain.net/smartcontract/dbs/event/blobber.go#L28) would allow us to test this endpoint GetBlobberIdsByUrlsHandler and getBlobbersForRequest using the benchmark program.